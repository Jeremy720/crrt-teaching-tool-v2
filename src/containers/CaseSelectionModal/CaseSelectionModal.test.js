import React from "react";
import { CaseSelectionModal } from "./";
import { shallow } from "enzyme";

describe("CaseSelectionModal", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<CaseSelectionModal />);
  });

  it("matches the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
