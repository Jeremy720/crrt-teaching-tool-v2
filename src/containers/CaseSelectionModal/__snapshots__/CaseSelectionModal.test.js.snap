// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaseSelectionModal matches the snapshot 1`] = `ShallowWrapper {}`;
